torchdata/datapipes/iter/util/dataframemaker.py

```diff
@@ -39,3 +41,40 @@ def __new__(
         batch_dp = source_dp.batch(dataframe_size)
         df_dp = batch_dp.map(partial(torcharrow.DataFrame, dtype=dtype, columns=columns, device=device))
         return df_dp
+
+
+@functional_datapipe("load_parquet_as_df")
+class ParquetDFIterDataPipe(IterDataPipe):  # IterDataPipe[torcharrow.IDataFrame[T_co]]
```

Comment:
`ParquetLoader`?

Reply:
And, I believe this class should be a StreamingDataFrame created from Task1.
cc: @VitalyFedyunin

Reply:
I updated the name to `ParquetDataFrameLoader` (because I want to emphasize that `DataFrame`s are the output. Let me know what the class should be.